release-notes/3.0/preview/api-diff/preview7/standalone-packages/3.0-preview7-standalone-packages_System.md

```diff
@@ -0,0 +1,176 @@
+# System
+
+``` diff
+ namespace System {
+-    public readonly struct Char8 : IComparable<Char8>, IEquatable<Char8> {
```

Comment:
Were these ever added in a previous previews api diff? If not, maybe we should remove this file. These types are not shipping as part of 3.0.

What changed that is causing this diff to show up?

cc @GrabYourPitchforks